palantirnet/search_api_federated_solr
# UMUS-31 Index Mapping

Creates a new "Federated Field" field for aggregating disparate entities into the same field in the solr index.

## To test
- Visit the fields list for your index (like https://labblog.uofmhealth.local/admin/config/search/search-api/index/federated_search_index/fields)
- Click "Add fields" then add a "Federated Field" <img width="701" alt="manage_fields_for_search_index_federated_measured_search_index___university_of_michigan_" src="https://user-images.githubusercontent.com/238201/35987659-ebd1bc14-0cc1-11e8-9f03-3b2a59f715b9.png">
- Edit the field label, machine name, and type if you like
- Click "Edit" to modify the field configuration, then add some config. You can either enter plain text or use the token browser to select tokens. <img width="733" alt="manage_fields_for_search_index_federated_measured_search_index___university_of_michigan_" src="https://user-images.githubusercontent.com/238201/35987841-755ad06a-0cc2-11e8-9d1e-b22af33a6b60.png">
- Notice that if you choose a token that's not available for the specific entity type (e.g. a "user" token on a "node") that you receive an error.
- Save your settings (in the config modal and on the fields page)
- Create/edit and save a piece of content to index it.
- Observe your fields are saved into the solr index using the proper data.
